tools/authors/add.php

```diff
@@ -201,7 +201,7 @@ function write_months_list($bd, $selected)
     }
     echo "</select>\n";
 }
-function write_days_list($bd, $selected)
+function write_days_list(string $bd, int $selected): void
 {
     echo "<select name=\"$bd" . "day\" size=\"1\">\n";
     echo "<option value=\"0\"" . ($selected == '0' ? ' SELECTED' : '') . ">" . _('Unknown');
```

Comment:
Note that we check against the string '0' using a non-strict equality on this line.

We could switch to a strict equality with 0 though. I punted on this as it seemed small (and PHP may already optimize this).